File: sub-model/src/main/java/org/github/base/mapper/ISysUserMapper.java
package org.github.base.mapper;

import org.github.base.entity.SysUserEntity;
import org.github.mybatis.MyBatisMapper;
import org.github.base.IMapper;

/**
 * <p>
 * 系统-用户表 Mapper 接口
 * </p>
 *
 * @author JYD_XL
 * @since 2019-04-14
 */
@MyBatisMapper
public interface ISysUserMapper extends IMapper<SysUserEntity> {}

File: sub-model/src/main/java/org/github/base/mapper/ISysUserMapper.kt
package org.github.base.mapper

import org.github.base.IMapper
import org.github.base.entity.SysUserEntity
import org.github.mybatis.MyBatisMapper

/**
 * <p>
 * 用户信息表 Mapper 接口
 * </p>
 *
 * @author JYD_XL
 * @since 2019-11-19
 */
@MyBatisMapper
interface ISysUserMapper: IMapper<SysUserEntity>
